docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -1301,9 +1328,9 @@ public class CustomJavaScriptRequestChecker implements JavaScriptRequestChecker
 }
 ----
 
-and reload the last requested page in case of a `499` status code.
+Reload the last requested page in case of a `499` status code.
```

Comment:
It kind of breaks the flow of the example `and reload` is correct IMHO

Reply:
Reverted these style changes back again.